.php_cs

```diff
@@ -0,0 +1,33 @@
+<?php
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__ . "/src");
+
+return PhpCsFixer\Config::create()
+    ->setRules(
+        [
+            '@PSR2'                               => true,
+            'array_syntax'                        => [
+                'syntax' => 'short'
+            ],
+            'class_keyword_remove'                => false,
+            'concat_space'                        => [
+                'spacing' => 'one'
+            ],
+            'combine_consecutive_unsets'          => true,
+            'general_phpdoc_annotation_remove'    => [
+                'annotations' => ['@author'],
+            ],
+            'linebreak_after_opening_tag'         => true,
+            'no_short_echo_tag'                   => true,
+            'no_useless_else'                     => true,
```

Comment:
The choice for using else everywhere was deliberate and is to improve readability. Please set this to false and re-add removed `else` statements.